07-routing-gossip.md

```diff
@@ -144,30 +144,50 @@ nodes for which a channel is not already known are ignored.
 2. data:
    * [64:signature]
    * [4:timestamp]
-   * [16:ipv6]
-   * [2:port]
    * [33:node-id]
    * [3:rgb-color]
    * [32:alias]
-   * [2:len]
-   * [len:features]
+   * [2:flen]
+   * [flen:features]
+   * [2:addrlen]
+   * [addrlen:addresses]
 
 The `timestamp` allows ordering in the case of multiple announcements;
-the `ipv6` and `port` allow the node to announce its willingness to
-accept incoming network connections, the `rgb-color` and `alias` allow
+the `rgb-color` and `alias` allow
 intelligence services to give their nodes cool monikers like IRATEMONK
 and WISTFULTOLL and use the color black.
 
+`addresses` allows the node to announce its willingness to accept
+incoming network connections: it contains series of `address
+descriptor`s for connecting to the node.  The first byte describes the
+address type, followed by the number of bytes for that type.
+
+The following `address descriptor` types are defined:
+
+1. `0`: padding.  data = none (length 0).
```

Comment:
This allows a node to bloat the announcement with padding, and every node will have to store it in case they'd like to forward it. That's also the reason why we dropped all paddings from BOLT07 and/or repurposed them.